src/modules/member/services/member.service.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { AuthService } from '@src/modules/core/auth/services/auth.service';
+import { PrismaService } from '@src/modules/core/database/prisma/prisma.service';
+import { MockPrismaService } from '@src/modules/test/mock-prisma';
+import { MockAuthService } from '@src/modules/test/mock-service';
+import { LoginByOAuthDto } from '../dtos/create-member-by-oauth.dto';
+import { MemberService } from './member.service';
+
+describe('MemberService', () => {
+  let memberService: MemberService;
+  let authService;
+  let prismaService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [
+        MemberService,
+        {
+          provide: PrismaService,
+          useValue: MockPrismaService,
+        },
+        {
+          provide: AuthService,
+          useValue: MockAuthService,
+        },
+      ],
+    }).compile();
+
+    memberService = module.get<MemberService>(MemberService);
+    authService = MockAuthService;
+    prismaService = MockPrismaService;
+  });
+
+  it('should be defined', () => {
+    expect(memberService).toBeDefined();
+    expect(authService).toBeDefined();
+    expect(prismaService).toBeDefined();
```

Comment:
```suggestion
    expect(memberService).toBeDefined();
```
memberService 가 의존하는 class 가 제대로 주입됐다면 memberService 가 정의 됐을테니 아래 두개는 확인하지 않아도 될거같은데 확인 한번 해주세요

Reply:
오 맞네요 expect(memberService).toBeDefined(); 가 성공적으로 이행 되면 memberService를 만드는데 필요한 class인 authService,prismaService는 확인해주지 않아도 되네요!! 수정하겠습니다!